mobile/lib/interfaces/album.interface.dart

```diff
@@ -2,6 +2,7 @@ import 'package:immich_mobile/entities/album.entity.dart';
 import 'package:immich_mobile/entities/asset.entity.dart';
 import 'package:immich_mobile/entities/user.entity.dart';
 import 'package:immich_mobile/interfaces/database.interface.dart';
+import 'package:immich_mobile/pages/albums/albums.page.dart';
```

Comment:
I think we don't want to import from pages in this layer. Can we move the QuickFterMode definition to either this file or a new 'model' file?